cmd/manager/main.go

```diff
@@ -181,7 +173,7 @@ func main() {
 	}
 
 	// Add the Metrics Service
-	addMetrics(ctx, cfg, namespace)
+	winmetrics.Add(ctx, cfg, namespace)
```

Comment:
Are we considering metrics an optional feature? Is that why we are not going to retry if we cant start metrics services?

Reply:
Metrics is dependent on the cluster having prometheus installed. So it is optional in that sense.